### PR TITLE
Fixed Potential Resource Busy Error

### PR DESCRIPTION
**Pull request by alxdb**
Calling `usb_device.set_configuration()` in `push2_python/display.py` can fail with `Resource Busy` if another program (potentially the kernel) has claimed one of the interfaces, since it attempts to reset the device. For example in Linux, Jack might claim one of the midi interfaces (I think this is what happened to me). Only the display interface needs to be claimed.

This is fixed by checking for an active configuration first, and only if it is not present calling `set_configuration`.

Also removed trailing whitespace in a few places, sorry about the diff, the only real change is in `push2_python/display.py`.

Thanks for writing this btw! Very useful :)

### CODE REVIEW

File: push2_python/constants.py [hunk 1]
```diff
@@ -9,8 +9,8 @@
 USB_TRANSFER_TIMEOUT = 1000
 
 # MIDI
-PUSH2_USER_PORT_NAME = 'Ableton Push 2 User Port'
-PUSH2_LIVE_PORT_NAME = 'Ableton Push 2 Live Port'
+PUSH2_USER_PORT_NAME = 'Ableton Push 2:Ableton Push 2 MIDI 1 28:0'
+PUSH2_LIVE_PORT_NAME = 'Ableton Push 2:Ableton Push 2 MIDI 2 28:1'
```
Comment (ffont) [inline push2_python/constants.py]: When I originally set these constants it was based on what's shown on macOS, but maybe other constants appear in other OSs. Maybe this should be done differently to make it work cross-platform. What OS are you using? I'd maybe revert this change here and open a new PR making these constants work cross-platform.

Comment (alxdb) [reply push2_python/constants.py]: I'm using linux, specifically I was using Jack. Yeah, I had to change these for my config, I forgot to change them back, sorry about that, I'll remove them. Not sure how to make this cross platform, not even sure it's possible, since it depends not only on the OS but on a bunch of other factors. Probably the best thing to do would be to make it configurable.

Comment (alxdb) [reply push2_python/constants.py]: have to make a new PR since I need these changes on my fork
